### DISCUSSION TITLE
V23 Release版本是否还支持debian源中的软件

### DISCUSSION THREAD

**Discussion by xingyankai** [linuxdeepin/developer-center#4676] [Development Questions | 开发答疑] [answered]
V23 Release做为根社区, 原来版本中支持的debian源中的软件, V23版本是全部废弃, 还是会继续兼容支持.

**Comment by BLumia** [comment, marked as answer]
不支持。切记，永远不要混源。即便是 v20 也不要混 debian 的源，因为 v20 仓库的一些底层包和 debian 也是有差异的。混源永远都是导致系统出问题的罪魁祸首。

如果真的要使用某些其它发行版源里提供的应用，建议考虑用 `distrobox`。可以参见 https://bbs.deepin.org/zh/post/257787 这个帖子。
